configure/ai-data-masking/get-started.mdx

```diff
@@ -29,6 +34,34 @@ Create an account at [Google Cloud Data Loss Prevention](https://cloud.google.co
 
 When installing hoop.dev, you need to set the environment variable `GOOGLE_APPLICATION_CREDENTIALS_JSON` with your GCP DLP credentials in hoop's Gateway. Hoop.dev uses Google Cloud's DLP at our protocol layer to mask sensitive data in real-time in the data stream of any connection you configure.
 
+### Host yourself with Microsoft Presidio
+
+<Steps>
+    <Step title="Install">
+        Check the Microsoft Presidio documentation to install it. We recommend you use the Docker Installation setup.
```

Comment:
same here, I would just give the snippets and link to source in case people need to understand of troubleshoot.

Reply:
I agree, also, in the card just below it it has the link for their docs so people will be directed there to check installation instructions